src/test/resources/application-test.yml

```diff
@@ -21,7 +22,6 @@ spring:
   main:
     allow-bean-definition-overriding: true
   redis:
-    host: localhost
     port: 6377
     password: ""
```

Comment:
i suspect these redis entries should be removed as have been superceeded by data.redis entries below

Reply:
TODO